modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -150,11 +179,72 @@ ServiceProvider setServiceProviderClaimConfig(ServiceProvider serviceProvider) {
     }
 
 	/**
-	 * Create Application with a given appDTO
+	 * Create Application with a given ApplicationModel
 	 *
-	 * @return OAuthConsumerAppDTO
-	 * @throws Exception
+	 * @param application application creation object
```

Comment:
Shall fix the formatting issues by adding missing full stops? Let's fix other places as well.

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096